pages/linux/lslocks.md

```diff
@@ -0,0 +1,19 @@
+# lslocks
+
+> List all local system locks.
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks -o PID,COMMAND,PATH`
+
+- List locks raw output (no columns), and no column headers:
+
+`lslocks -r -n`
+
+- Populate a csv file with the information from above command:
```

Comment:
I don't think this example is needed. After all it's just the same as the previous one with an added pipe which only replaces spaces with commas (also, a CSV doesn't realy *need* to be comma-delimited, CSVs can be space-delimited too). It does not really show different features of the `lslocks` command.

I would suggest adding examples for the `--pid` and `--json` options instead, which are not mentioned. If you don't feel like or don't have the time to add examples for those too, that's fine they can be added in the future, but I wouldn't populate the page with more complex examples without having all the basic functionality covered first.

Reply:
Ok, will remove this example ( I only added that as I used it myself quite recently ) and add the `--pid` and `--json` examples after work this evening. Thanks for feedback 👍